packages/block-library/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Master (unreleased)
 
+### Enhancements
+
+- Heading block uses `has-text-align-*` class names rather than inline style for text alignment.
```

Comment:
Why is this here?

Reply:
It was missed in other PR, I didn't want to bother myself with another PR and going through the review process which takes ages.